sos/report/plugins/kubernetes.py

```diff
@@ -85,15 +96,16 @@ def setup(self):
         for subcmd in ['version', 'config view']:
             self.add_cmd_output('%s %s' % (self.kube_cmd, subcmd))
 
-        # these are not namespaced, must pull separately.
-        global_resources = [
-            'namespaces',
-            'projects',
-            'pvs'
-        ]
-        self.add_cmd_output([
-            "%s get %s" % (self.kube_cmd, res) for res in global_resources
-        ])
+        if self.get_option('all'):
+            self.add_cmd_output([
+                "%s get -o json %s" % (self.kube_cmd, res)
+                for res in self.global_resources
+            ])
+        else:
+            self.add_cmd_output([
+                "%s get %s" % (self.kube_cmd, res)
+                for res in self.global_resources
```

Comment:
f-strings instead of string substitution, please.

Reply:
this is changed